content/adjustable_income.md

```diff
@@ -3,6 +3,7 @@ label: Get an adjustable income
 description: Your money is invested in a flexi-access drawdown fund to give you a regular income with the option to take cash when you need to.
 tags:
   - option
+  - next-and-previous
```

Comment:
Kinda feels to me like we're starting to misuse tags. Maybe this should be promoted to an actual front matter key like `paginates: true` or something similar?